### PR TITLE
Remove deprecated function calls

### PR DESCRIPTION
**Pull request by antograssiot**
Just remove some function calls that have been deprecated recently from the core

### CODE REVIEW

File: src/TestSuite/Stub/Response.php [hunk 1]
```diff
@@ -29,7 +29,7 @@ class Response extends Base
     public function send()
     {
         if ($this->hasHeader('Location') && $this->_status === 200) {
-            $this->statusCode(302);
+            $this->withStatus(302);
```
Comment (dereuromark) [inline src/TestSuite/Stub/Response.php]: Wouldnt you need to assign the immutable response?

**Review by markstory — COMMENTED** [review]
These changes might have already been applied to 3.next.

**Comment by antograssiot** [comment]
Oh sorry @markstory, I'll rebase onto 3.next to see if  some changes are still needed and then update the target branch of this PR or just close it

File: src/Console/CommandRunner.php [hunk 2]
```diff
@@ -31,7 +31,7 @@
  */
 class CommandRunner
```
Comment (robertpustulka) [inline src/Console/CommandRunner.php]: Could you add a `EventDiapatcherInterface` here?

**Pull request merged**